shesha/util/make_pupil.py

```diff
@@ -213,6 +211,9 @@ def make_VLT(dim, pupd, tel):
         tel: (Param_tel) : Telescope structure
     """
 
+    if(tel.cobs == -1):
```

Comment:
I am not a big fan of this convention...
I agree it is similar to t_spiders, but cobs is not seen as an optional parameter, so its value is used without checks elsewhere in the code, and putting it at -1 by default can impact other part of the code...
We have to double check that